actionpack/lib/action_controller/metal/instrumentation.rb

```diff
@@ -35,6 +35,9 @@ def process_action(*)
         payload[:response] = response
         payload[:status]   = response.status
         result
+      rescue => error
+        payload[:status] = ActionDispatch::ExceptionWrapper.status_code_for_exception(e.class.name)
```

Comment:
```suggestion
        payload[:status] = ActionDispatch::ExceptionWrapper.status_code_for_exception(error.class.name)
```

Reply:
Tiny mistake :/

Reply:
Yeah your mercy was too exciting for me ;-)